### PR TITLE
Refactor components

### PR DESCRIPTION
**Pull request by henryzarza**
## Summary

- Refactor components.
- Corrections in styles.
- Apply `card` component to components. 

## Screenshots

refactor

### CODE REVIEW

File: src/components/Button/index.scss [hunk 1]
```diff
@@ -0,0 +1,32 @@
+@import './scss/btn-1';
+@import './scss/btn-2';
+@import './scss/btn-3';
+@import './scss/btn-4';
+@import './scss/btn-5';
+
+button.btn,
```
Comment (FrankIglesias) [inline src/components/Button/index.scss]: can we delete this tag?

File: src/components/ProgressBar/index.scss [hunk 2]
```diff
@@ -1,53 +1,27 @@
...
-  </div>
-</template>
-<style lang="scss" scoped>
-:root {
-  --percentageBar: 90%;
+@import '../scss/variables/_sizes';
 
+.progress {
+  --percentageBar: 30%;
+
```
Comment (FrankIglesias) [inline src/components/ProgressBar/index.scss]: delete thsi extra spaces

File: src/components/ProgressBar/index.scss [hunk 3]
```diff
@@ -1,53 +1,27 @@
...
-    </div>
-  </div>
-</template>
-<style lang="scss" scoped>
-:root {
-  --percentageBar: 90%;
+@import '../scss/variables/_sizes';
 
+.progress {
+  --percentageBar: 30%;
```
Comment (FrankIglesias) [inline src/components/ProgressBar/index.scss]: use spinal case for variables if you can

File: src/components/RadioButtons/index.scss [hunk 4]
```diff
@@ -0,0 +1,99 @@
+@import '../scss/variables/colors';
+
+.item {
+  align-items: center;
+  display: flex;
+  justify-content: space-evenly;
+}
+
+.circle-progress-radio svg {
```
Comment (FrankIglesias) [inline src/components/RadioButtons/index.scss]:
```scss
.circle-progress-radio svg {
  circle {
   ....
   }
  path {
  ....
  }
}
```

File: src/views/index/index.js [hunk 5]
```diff
@@ -2,14 +2,16 @@ import Vue from 'vue'
 
 import { installServiceWorker } from '../../serviceWorkerInstaller'
 
-const ButtonGroup = () => import(/* webpackChunkName: "example" */ '../../components/Button')
-const CheckboxGroup = () => import(/* webpackChunkName: "example" */ '../../components/Checkbox')
-const ProgressBar = () => import(/* webpackChunkName: "example" */ '../../components/ProgressBar')
-const RadioButtons = () => import(/* webpackChunkName: "example" */ '../../components/RadioButtons')
+const ButtonGroup = () => import(/* webpackChunkName: "example" */ '../../components/Button/index')
```
Comment (FrankIglesias) [inline src/views/index/index.js]: why did you choose to split some components into three files? 
'Cause of the file size?

Comment (henryzarza) [reply src/views/index/index.js]: Yes, for that reason.

**Pull request closed without merging**